src/piece_picker.cpp

```diff
@@ -2078,7 +2083,7 @@ namespace {
 		{
 			if (m_dirty) update_pieces();
 			TORRENT_ASSERT(!m_dirty);
-
+            //The piece priority logic may be removed in the future.
```

Comment:
this should be removed

Reply:
Ok. Don't forget about it in the future.

Reply:
if you think the piece priority feature should be removed, please start a separate thread/ticket where you present your arguments

Reply:
@arvidn Argument 1: no usage scenarios.

Reply:
please start a separate thread/ticket where you present your arguments